run_brer/run_config.py

```diff
@@ -377,8 +377,12 @@ def __train(self, tpr_file=None, **kwargs):
             # of alpha.
             # noinspection PyUnresolvedReferences
             current_alpha = context.potentials[i].alpha
+            if current_alpha == 0:
+                self._logger.error('Alpha value was constrained to 0.0')
```

Comment:
Shouldn't this produce an actual error (throw an exception) instead of just logging?